lib/plugins/aws/invokeLocal/index.js

```diff
@@ -467,7 +467,7 @@ class AwsInvokeLocal {
           ]);
 
           const dockerArgsFromOptions = this.getDockerArgsFromOptions();
-          const dockerArgs = _.concat(
+          const dockerArgs = [].concat(
             ['run', '--rm', '-v', `${artifactPath}:/var/task:ro,delegated`],
```

Comment:
Let's use `concat` directly on first array instead (we do not have to create an empty array in this case)